Regression/WarpX-tests.ini

```diff
@@ -936,6 +936,23 @@ analysisRoutine = Examples/Tests/langmuir/analysis_rz.py
 analysisOutputImage = Langmuir_multi_rz_analysis.png
 aux1File = Regression/PostProcessingUtils/post_processing_utils.py
 
+[laser_envelope_model]
+buildDir = .
+inputFile = Examples/Tests/laser_envelope_model/input_laser_envelope_model
+runtime_params =
+dim = 2
+addToCompileString =
+cmakeSetupOpts = -DWarpX_DIMS=2 -DCMAKE_BUILD_TYPE=Debug
```

Comment:
We can keep the compiler flag `-DCMAKE_BUILD_TYPE=Debug` for testing purposes for now, but will have to remove it in the end for performance consistency within CI.